protocol/x/vest/types/keys.go

```diff
@@ -15,6 +15,9 @@ const (
 
 	// VestEntryKeyPrefix is the prefix used when storing a VestEntry in the state.
 	VestEntryKeyPrefix = "vest_entry"
+
+	// CommunityTreasury defines the root string for community treasury module account.
+	CommunityTreasury = "community_treasury"
```

Comment:
Can you add a vester account for `community_vester` too?

Reply:
Sure! Is it mainly to be consistent with the `vester` / `treasury` pattern?

Reply:
Yeah, there's always a vester/treasury pair for these accounts AFAIK. Same for V3 as well.